src/win32/do_upgrade.ps1

```diff
@@ -101,26 +81,10 @@ write-output "$(Get-Date -format u) - Reading status file: $($status)" >> .\upgr
 If ($status -eq $null)
 {
     write-output "2" | out-file ".\upgrade\upgrade_result" -encoding ascii

```

Comment:
You can add a log saying that the upgrade failed and Wazuh will stay in the state it is at this point.